### PR TITLE
Create RoleType enum

### LINKED ISSUE #55: Create `RoleType` enum
Professor / Tutor / Student

### PR DESCRIPTION
**Pull request by MAOXIONGKAI**
- [ ] Enum name follows naming convention specified in the course material
- [ ] Supports value specified in the requirement(STUDENT, TUTOR, PROFESSOR)
- [ ] Passes all the checkstyle requirements(Javadoc, indentation etc.)
- [ ] Created under person package because it is tagged under person

Closes #55

### CODE REVIEW

**Comment by MAOXIONGKAI** [comment]
It seems like I can't merge this directly after any approval because the new code added decreased the coverage level of the existing test cases, but I have found similar issues[Codecov test failing](https://github.com/nus-cs2103-AY2425S1/forum/issues/325) and according to the prof response, we can try resolve this by:
1. Decrease the the code coverage limit, which probably need someone with write access to help(@weiseng18)
2. Write test cases to cover this part of the code(but I don't think there's anyway to do that, since there's nothing to test).

So, I am proposing a decrease of coverage limit to pass this PR(Since this week is more about tweaking the code instead of actually implementing the functions, writing test cases may not be possible and this issue may easily be triggered by others as well)

File: src/main/java/seedu/address/model/person/RoleType.java
Comment (lithops-zty) [inline src/main/java/seedu/address/model/person/RoleType.java:11]:
```suggestion
     * Represents a student studying under a specific course
```

File: src/main/java/seedu/address/model/person/RoleType.java
Comment (lithops-zty) [inline src/main/java/seedu/address/model/person/RoleType.java:16]:
```suggestion
     * Represents a tutor teaching/mentoring tutorials/labs of a course
```

File: src/main/java/seedu/address/model/person/RoleType.java
Comment (lithops-zty) [inline src/main/java/seedu/address/model/person/RoleType.java:21]:
```suggestion
     * Represents a professor who is in charge of teaching/managing the course
```

**Review by lithops-zty — CHANGES_REQUESTED** [review]
LGTM! Except you may want to take a look at singular/plural inconsistencies in the JavaDoc.

**Pull request merged**
